test/integration/analysis-controller/00-teststep-template.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  #  substitutes current time and namespace, making sure they are changed to env var first
+  # to prevent bad files in case of a test interrupt
+  - script: |
+      current_time=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+      initial_timestamp=$(date -u -d "$current_time" +%s)
+      seconds_to_add=20
+      new_timestamp=$((initial_timestamp + seconds_to_add))
+      time_20_seconds_later=$(date -u -d "@$new_timestamp" +"%Y-%m-%dT%H:%M:%SZ")
+      export CURRENT_TIME="$current_time"
+      export LATER="$time_20_seconds_later"
+      sed -i '0,/^  namespace:/s/namespace: .*/namespace: $NAMESPACE/' install.yaml
+      sed -i 's/from: .*$/from: $CURRENT_TIME/g; s/to: .*$/to: $LATER/g' install.yaml
```

Comment:
let's try to use envsubst https://www.baeldung.com/linux/envsubst-command

Reply:
good suggestion, will adapt that